src/jit/flowgraph.cpp

```diff
@@ -7861,8 +7861,8 @@ void Compiler::fgAddSyncMethodEnterExit()
 #ifdef DEBUG
         if (verbose)
         {
-            printf("\nSynchronized method - Add 'acquired' initialization in first block BB%02u [%08p]\n", fgFirstBB,
-                   dspPtr(fgFirstBB));
+            printf("\nSynchronized method - Add 'acquired' initialization in first block BB%02u [%04u]\n", fgFirstBB,
```

Comment:
Looks like an existing bug here where the first arg should be `fgFirstBB->bbNum`